bin/test-workflow/0_prep_env.sh

```diff
@@ -22,6 +22,9 @@ else
   PLATFORM="kubernetes"
 fi
 export PLATFORM
+if [[ "$CONJUR_PLATFORM" == "kind" ]]; then
+    RUN_LOCAL_CONTAINER="false"
```

Comment:
Your call, but I wonder if it would be clearer or more obvious if we had the word `CLIENT` in this variable name (e.g. `RUN_CLIENT_CONTAINER` or `LOCAL_CLIENT_CONTAINER` or `RUN_CLIENT_CONTAINER`)? Wasn't sure while reading this what this container was for.